lib/pg_ha_migrations/struct.rb

```diff
@@ -0,0 +1,59 @@
+module PgHaMigrations::Struct
+  Table = Struct.new(:name, :schema) do
+    delegate :inspect, to: :name
+
+    def fully_qualified_name
+      @fully_qualified_name ||= [
+        ActiveRecord::Base.connection.quote_schema_name(schema),
+        ActiveRecord::Base.connection.quote_table_name(name),
+      ].join(".")
+    end
+
+    def present?
+      name.present? && schema.present?
+    end
+  end
+
+  Index = Struct.new(:name, :table) do
```

Comment:
I think Struct::Table and Struct::Index should maybe be the same thing. Or, at the very least, table should just be Relation, and Index should be a subclass of that.

Reply:
Addressed in https://github.com/braintree/pg_ha_migrations/pull/87/commits/bb4b0ce47c6e88de1a0f96381583efa72a8b4495